tests/libcxx/run_tests_locally.sh

```diff
@@ -16,29 +16,18 @@ FILE=$1
 while read test; do
   echo "$test"
   OUTPUT=$(2>&1 timeout 10 make one TEST=$test )
+  RETURN_VAL=$?
   echo $OUTPUT >> temp_$FILE.output
-  HAS_UNHANDLED_SYSCALL=$(2>&1 timeout 10 make one TEST=$test | grep "unhandled")
-  if [ -z "$HAS_UNHANDLED_SYSCALL" ]
+  TERMINATED=$(echo "$OUTPUT" | grep -F 'terminate called') # check for termination due to exceptions
```

Comment:
Could you clarify what case this covers? Can the return code ever be zero in this case?

Reply:
This covers the case in which libcxx generates an exception and calls terminate on it. mystikos takes the exception literally, so it seems to have failed, but the error code is still 0. Until we fix the cpp exception handling, these tests would belong in the failed category

Reply:
eg: 
```
/app/llvm-project/libcxx/test/libcxx/containers/associative/map/at.abort.pass.cpp.exe terminate called after throwing an instance of 'std::out_of_range' what(): map::at
```

Reply:
Digging further, it seems to me this test is doing the right thing. 
Looks like its built with no-exceptions - https://github.com/salsal97/llvm-project/blob/main/libcxx/test/libcxx/containers/associative/map/at.abort.pass.cpp#L18
According to [this SO answer](https://stackoverflow.com/a/7249442/4307215), with no-exceptions, throws are converted into `abort()` calls. (The test checks for this in the SIGABRT handler - `exit_success`)

Reply:
So should this be considered as a passing test? Its not explicitly built with no-exceptions [here](https://github.com/deislabs/mystikos/blob/main/tests/libcxx/Dockerfile#L21) but it might be the default behavior. Is there a flag that'll enable graceful exception handling in that case?

Reply:
> So should this be considered as a passing test? 
I think so. One way to confirm would be to compare with running on Alpine Linux.

Reply:
From https://www.cs.colostate.edu/~cs253/Spring19/Lecture/ExceptionHandling -

If something is thrown but never caught, then the special function terminate() is called, which complains in an implementation-defined manner, and stops the program by calling abort().

Looks like the `no-exceptions` line in the source file is a red-herring.

>  Is there a flag that'll enable graceful exception handling in that case?

In this case, I think the exception is handled the right way. The throwing of the exception(due to L34) causes an SIGABRT, which is handled by the `exit_success` handler, and returns exit code 0.

Reply:
You're right, I understand now. Expecting no output from this test is not the desirable result. I've updated the PR to remove this line from the script and added the exception tests to the pipeline